modules/openxBidAdapter.js

```diff
@@ -248,6 +248,24 @@ function buildCommonQueryParamsFromBids(bids, bidderRequest) {
     nocache: new Date().getTime()
   };
 
+  const firstPartyData = config.getConfig('ortb2.user.data')
+  if (Array.isArray(firstPartyData) && firstPartyData.length > 0) {
+    const sm = firstPartyData
+      .filter(
+        data => (Array.isArray(data.segment) &&
+                data.segment.length > 0 &&
+                data.name !== undefined &&
+                data.name.length > 0)
+      )
+      .map(
+        data => data.name + ':' + data.segment.map(seg => seg.id).join('|')
```

Comment:
you appear to drop segtax, which is critical to understand the segment id.

Reply:
Thank you, the provider we were interested in (permutive) appears to generate proprietary segments but since the code is generic I added handling for segtax as well.